src/SME.SGP.Aplicacao/Queries/Turma/VerificaPodePersistirTurmaDisciplina/VerificaPodePersistirTurmaDisciplinaQuery.cs

```diff
@@ -9,20 +9,22 @@ namespace SME.SGP.Aplicacao
 {
     public class VerificaPodePersistirTurmaDisciplinaQuery : IRequest<bool>
     {
-        public VerificaPodePersistirTurmaDisciplinaQuery(Usuario usuario, string turmaId, string componenteCurricularId, DateTime data, bool historico)
+        public VerificaPodePersistirTurmaDisciplinaQuery(Usuario usuario, string turmaId, string componenteCurricularId, DateTime data, bool historico, bool ehTerritorioSaber)
```

Comment:
acho que um parâmetro opcional ficaria legal pois caso tenha mais outros pontos que utilizem esse query mediator, já não necessita forçar o "false" ou "true", dai já daria para deixar padronizado como "bool ehTerritorioSaber = false", o que acha?

Reply:
verificando

Reply:
ajustado